### PR TITLE
Wire up some enzyme tests

### PR DESCRIPTION
**Pull request by tough-griff**
Add some simple and a few not-so-simple enzyme specs for pages.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -13,6 +13,6 @@ export * from './settings/Profile';
 export * from './Dashboard';
 export * from './Templates';
 export * from './reports';
-export { default as AuthPage } from './AuthPage';
+export * from './auth'; // TODO
```
Comment (tough-griff) [inline src/pages/index.js]: This will match #78

File: src/pages/auth/AuthPage.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import styles from './AuthPage.module.scss';
+
+export class AuthPage extends Component {
+  state = {
+    username: '',
+    password: '',
+    rememberMe: false
+  };
+
+  onChangeUsername = (evt) => {
```
Comment (tough-griff) [inline src/pages/auth/AuthPage.js]: Refactored so we're not instantiating new functions on each render.

File: src/pages/auth/tests/AuthPage.snapshot.test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { AuthPage } from '../AuthPage';
+import { props } from './AuthPage.test';
```
Comment (tough-griff) [inline src/pages/auth/tests/AuthPage.snapshot.test.js]: Do we want snapshot tests in a separate file?
Do we want to snapshot test each case (as tested against in `AuthPage.test.js`)?

File: src/pages/auth/AuthPage.js [hunk 4]
```diff
@@ -8,86 +8,97 @@ import { Panel, Button, TextField, Checkbox } from '@sparkpost/matchbox';
...
       </div>
     );
   }
 
-  renderLoginButton() {
-    return this.props.auth.loginPending ? <span><i className="fa fa-spinner fa-spin"></i> Logging In</span> : <span>Log In</span>;
+  renderLoginButtonText() {
+    return this.props.auth.loginPending
+      ? <span>
+          <i className="fa fa-spinner fa-spin" /> Logging In
```
Comment (jonambas) [inline src/pages/auth/AuthPage.js]: Heh

**Pull request merged**